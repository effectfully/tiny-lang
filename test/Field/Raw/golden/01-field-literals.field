/**
  * Parsing field literals
  */
let zero  : field = 0;
let one   : field = 1;
let two   : field = 2;
let half  : field = 1/2;
let third : field = 1/3;
