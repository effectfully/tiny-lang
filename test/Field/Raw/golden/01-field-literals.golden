Program [] Statements [ELet (Var "zero") (EConst (Some 0)),ELet (Var "one") (EConst (Some 1)),ELet (Var "two") (EConst (Some 2)),ELet (Var "half") (EAppBinOp Div (EConst (Some 1)) (EConst (Some 2))),ELet (Var "third") (EAppBinOp Div (EConst (Some 1)) (EConst (Some 3)))]